event_survey/wizard/event_survey_generator_view.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+  <record id="event_survey_generate_view" model="ir.ui.view">
+    <field name="name">event.survey.generator form</field>
+    <field name="model">event.survey.generator</field>
+    <field name="arch" type="xml">
+      <form string="Create Survey">
+        <group>
+          <field name="survey_template_id"
```

Comment:
Not mandatory, can we add `event_id` in readonly here? With these kind of wizards I like to display the record we are going to process as the pop-up hide the current form. If the readonly makes trouble (maybe the value won't be sent to the controller), we can now use the new `force_save: True` context key :)

Reply:
don't think so, works properly, use key just to try :) 
and also can create template if we need

Reply:
I was speaking about adding a new `<field name="event_id"/>` but it was only a suggestion, not blocking.